src/ucp/wireup/address.c

```diff
@@ -684,37 +996,73 @@ ucp_address_unpack_iface_length(ucp_worker_h worker, const void *flags_ptr,
 
     if (!is_ep_addr) {
         *is_last_iface = *(uint8_t*)ptr & UCP_ADDRESS_FLAG_LAST;
+        extendable     = addr_version != UCP_ADDRESS_VERSION_V1;
```

Comment:
pass version directly to ucp_address_unpack_byte_extended

Reply:
but for ep addr I do not need to extend the length even for version2